README.md

```diff
@@ -125,4 +126,25 @@ profile dimension and suite name as the "grep" dimension. Giving 2x2=4 parallel
 Since the stdout output is coming to the parent process as it happens, it is most useful to incorporate a reporter which
 can output a separate file per parallel instance. Try using "mochawesome" for that. You will find that nemo-runner is
 already set up to use mochawesome reports and give them an appropriate filename. [Please have a look at the nemo-example-app
-for a full example using "mochawesome"](https://github.com/paypal/nemo-example-app/blob/nemo-3-alpha/test/functional/config/profiles.json).
\ No newline at end of file
+for a full example using "mochawesome"](https://github.com/paypal/nemo-example-app/blob/nemo-3-alpha/test/functional/config/profiles.json).
+
+### Mocha options
+
+The properties passed in to the `"mocha"` property of `config.json` will be applied to the `mocha` instances that are created. In general, these properties correlate with the `mocha` command line arguments. E.g. if you want this:
```

Comment:
Important not to give the impression that all the CLI args are available. It's only those listed in the "programmatic use of mocha" doc: https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options